packages/@aws-cdk/toolkit-lib/lib/api/drift/drift-formatter.ts

```diff
@@ -114,6 +111,7 @@ export class DriftFormatter {
         numResourcesWithDrift: 0,
         numResourcesUnchecked: this.allStackResources.size - this.resourceDriftResults.length,
         stackHeader,
+        unchecked: formatterOutput.unchecked,
```

Comment:
Include unchecked resources even when the count of drifted resources is 0, for the user's awareness.